js/src/button.js

```diff
@@ -167,6 +170,33 @@ $(document)
     $(button).toggleClass(ClassName.FOCUS, /^focus(in)?$/.test(event.type))
   })
 
+$(window).on(Event.LOAD_DATA_API, () => {
+  // ensure correct active class is set to match the controls' actual values/states
+
```

Comment:
trailing line

Reply:
wonder if that could be left there, as it makes it clearer that there's then two separate blocks that follow (otherwise the comment on the next line sort of merges with this one?

Reply:
Yep you can merge with the other one seems fine to me

Reply:
do you mean yep can leave that extra line, or "no, remove the line"?

it's a small thing of course, but feel that `// ensure correct active class is set to match the controls' actual values/states` is the comment/description of what happens in the whole thing, and then `// find all checkboxes/readio buttons inside data-toggle groups` and `// find all button toggles` are subsections...so it's visually clearer if there's that empty line

Reply:
or was it a "you can merge the PR"?